src/Resources/private/default-gift-card-configuration-css.txt

```diff
@@ -0,0 +1,10 @@
+body {
```

Comment:
Why not name this file `.css`?

Reply:
To make it clear that it is no CSS loaded the regular way

Reply:
Should this file be a `.css.twig` file and be rendered using twig? Wdyt?

Reply:
Absolutely. And it simplifies some parts also :)